### DISCUSSION TITLE
Attempting to enable SSL/TLS cert on SecOnion SOC UI

### DISCUSSION THREAD

**Discussion by PembertonBoys** [Security-Onion-Solutions/securityonion#9964] [General]
Hello,

I am new to Security Onion and it has been a very frustrating process of learning with a major lack of resources for support.  I have learned quite a bit and have managed to stand up an architecture with a master and a few sensor servers but I am attempting to install a new certificate to my UI but after I copied over the new cert & key, per the Danny V page everyone uses, I get a blank screen when attempting to access the UI.  I don't even know where to turn to for further troubleshooting.  When I look in the "messages" log file I see one thing that stands out but I don't even know if this is related:

**Mar 20 17:09:05 seconion-master launcher: {"caller":"request_queries.go:162","err":"rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: Error while dialing dial tcp <IP address removed>:8090: connect: connection refused\"","method":"RequestQueries","reauth":false,"res":"null","severity":"info","took":"88.803µs","ts":"2023-03-20T17:09:05.761833617Z","uuid":"120450cf-1fa4-495b-8c80-97e7e4ef69de"}**

I guess I am looking for advice on where I should check for troubleshooting to see if my server is configured properly.  I appreciate any advice anyone can shed on this issue.  Thank you.

**Comment by cm-ops** [comment]
Can you let me know what your steps were to do this? So you see any issues in the SOC log `/opt/so/log/soc/sensoroni-server.log`?
